hassio/src/hassio-main.ts

```diff
@@ -143,7 +124,7 @@ class HassioMain extends ProvideHassLitMixin(HassRouterPage) {
       });
     });
 
-    makeDialogManager(this, document.body);
+    makeDialogManager(this, this);
```

Comment:
This should be done differently as it now interferes with the logic of `HassRouterPage` to add elements, we should probably add an element in between, like `<hassio>`: handles dialogs -> `<hassio-main>`: router